content/en/docs/reference/command-line-tools-reference/feature-gates/pod-lifecycle-sleep-action-allow-zero.md

```diff
@@ -0,0 +1,14 @@
+---
+title: PodLifecycleSleepActionAllowZero
+content_type: feature_gate
+_build:
+  list: never
+  render: false
+
+stages:
+  - stage: alpha 
+    defaultValue: false
+    fromVersion: "1.32"
+    toVersion: "1.32"
```

Comment:
(nit)
```suggestion
```

(because of https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future - `toVersion: 1.32` means: this gate was no longer alpha in v1.33)